### DISCUSSION TITLE
ıbxc listelenme

### DISCUSSION THREAD

**Discussion by Kriptoportakal** [IBAX-io/go-ibax#659] [Soru Cevap - Türkçe]
1 şubatta hangi borsalarda listelenme bekliyorsunuz?

**Comment by Yusufknk** [comment]
Coinmarketcap ve Coingecko'da listelenecektir. Ön satış ve borsa işlemleri ilerleyen zamanlarda sosyal medya kanallarında duyurulacaktır.

**Reply by coinsinov** [reply]
Merakla bekliyoruz ;)

**Comment by satocn** [comment]
Bakalım umarım tutar.
